SETUP/smoketests/pageload-smoketest.sh

```diff
@@ -0,0 +1,253 @@
+#!/bin/bash
```

Comment:
For shell scripts, I've been running them through shell check (https://www.shellcheck.net/) and learning a ton about shell scripts. Not necessary that you address these, but there is some good insight in the output:

```
[Line 21:](javascript:setPosition(21, 1))
DP_USER=admin
^-- [SC2209](https://www.shellcheck.net/wiki/SC2209) (warning): Use var=$(command) to assign output (or quote to assign string).
 
[Line 196:](javascript:setPosition(196, 5))
    ${NOLOGIN_URLS[*]}
    ^-- [SC2206](https://www.shellcheck.net/wiki/SC2206) (warning): Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
 
[Line 197:](javascript:setPosition(197, 5))
    ${BASE_URLS[*]}
    ^-- [SC2206](https://www.shellcheck.net/wiki/SC2206) (warning): Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
 
[Line 198:](javascript:setPosition(198, 5))
    ${API_URLS[*]}
    ^-- [SC2206](https://www.shellcheck.net/wiki/SC2206) (warning): Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
 
[Line 199:](javascript:setPosition(199, 5))
    ${FAQ_URLS[*]}
    ^-- [SC2206](https://www.shellcheck.net/wiki/SC2206) (warning): Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
 
[Line 200:](javascript:setPosition(200, 5))
    ${MISC_URLS[*]}
    ^-- [SC2206](https://www.shellcheck.net/wiki/SC2206) (warning): Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
 
[Line 201:](javascript:setPosition(201, 5))
    ${QUIZ_URLS[*]}
    ^-- [SC2206](https://www.shellcheck.net/wiki/SC2206) (warning): Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
 
[Line 202:](javascript:setPosition(202, 5))
    ${TEAMS_URLS[*]}
    ^-- [SC2206](https://www.shellcheck.net/wiki/SC2206) (warning): Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
 
[Line 203:](javascript:setPosition(203, 5))
    ${STATS_URLS[*]}
    ^-- [SC2206](https://www.shellcheck.net/wiki/SC2206) (warning): Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
 
[Line 204:](javascript:setPosition(204, 5))
    ${STYLES_URLS[*]}
    ^-- [SC2206](https://www.shellcheck.net/wiki/SC2206) (warning): Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
 
[Line 205:](javascript:setPosition(205, 5))
    ${BASE_TOOL_URLS[*]}
    ^-- [SC2206](https://www.shellcheck.net/wiki/SC2206) (warning): Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
 
[Line 219:](javascript:setPosition(219, 20))
    /bin/kill -INT $(jobs -p)
                   ^-- [SC2046](https://www.shellcheck.net/wiki/SC2046) (warning): Quote this to prevent word splitting.
 
[Line 229:](javascript:setPosition(229, 24))
if [ $CACHE_COOKIE = 1 -a -f .smoketest-cookie ]; then
                       ^-- [SC2166](https://www.shellcheck.net/wiki/SC2166) (warning): Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.
 
[Line 239:](javascript:setPosition(239, 35))
    [ $CACHE_COOKIE = 1 ] && echo $COOKIE > .smoketest-cookie
                                  ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([13])), apply [all SC2086](javascript:applyFixCode(2086)))
    [ $CACHE_COOKIE = 1 ] && echo "$COOKIE" > .smoketest-cookie
 
[Line 242:](javascript:setPosition(242, 12))
for URL in ${URLS[@]}; do
           ^-- [SC2068](https://www.shellcheck.net/wiki/SC2068) (error): Double quote array expansions to avoid re-splitting elements.
 
[Line 244:](javascript:setPosition(244, 18))
    echo Getting $URL
                 ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([15])), apply [all SC2086](javascript:applyFixCode(2086)))
    echo Getting "$URL"
 
[Line 248:](javascript:setPosition(248, 25))
        http://$IP_PORT/$URL > $OUT
                        ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([16])), apply [all SC2086](javascript:applyFixCode(2086)))
        http://$IP_PORT/"$URL" > $OUT

$
```

Reply:
Done.